.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,24 @@
+# Workflow name
+name: 'Chromatic Deployment'
+
+# Event for the workflow
+on: push
```

Comment:
i was originally doing it on PR request, but I think on push is fine and that's what they recommend for some git history thing